lessons/05-html-linking/news.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <title>News</title>
+    </head>
+    <body>
+        <h1>News</h1>
+<nav>
+    <ul>
+        <li><a href="./index.html"> Home</a></li>
+        <li><a href="./news.html"> News</a></li>
+        <li><a href="./about.html"> About</a></li>
+        <li><a href="contact.html"> Contacts</a></li>
+    </ul>
+    <a href="https://thestarphoenix.com/" target="_blank"> Go to The StarPhoenix</a>
+</nav>
```

Comment:
This `<a>` tag should be outside of the `<nav>`. We want the contents of the nav to be how we move around our site